.storybook/stories/tutorials/ServerSideRendering/story/index.jsx

```diff
@@ -0,0 +1,304 @@
+import PicassoBook from '~/.storybook/components/PicassoBook'
+
+const page = PicassoBook.section('Tutorials').createPage(
+  'Server Side Rendering and Picasso',
```

Comment:
```suggestion
  'Server Side Rendering',
```
should be enough I think